ddtrace/appsec/ddwaf/__init__.py

```diff
@@ -24,11 +24,12 @@
     from .ddwaf_types import ddwaf_get_version
     from .ddwaf_types import ddwaf_init
     from .ddwaf_types import ddwaf_object
+    from .ddwaf_types import ddwaf_result
     from .ddwaf_types import ddwaf_result_free
     from .ddwaf_types import ddwaf_ruleset_info
+    from .ddwaf_types import ddwaf_run
     from .ddwaf_types import ddwaf_update_rule_data
     from .ddwaf_types import py_ddwaf_required_addresses
-    from .ddwaf_types import py_ddwaf_run
```

Comment:
Since this is a public module removing this import is a "breaking change". The impact should be small so I think it should be fine.